versioned_docs/version-1.3.0-beta.1/installation.mdx

```diff
@@ -27,6 +27,20 @@ snap install oras --classic
 
 Install ORAS from the latest [release artifacts](https://github.com/oras-project/oras/releases):
 
+In most cases, you can determine operating system and architecture using `uname`:
```

Comment:
I think `uname` is not applicable in Windows (PowerShell). Should we state that these commands are for Unix-based systems (Linux, macOS) ?

Reply:
```suggestion
For macOS and Linux type operating systems, you can determine operating system and architecture using `uname`:
```